centreon/www/class/centreonACL.class.php

```diff
@@ -1715,32 +1715,63 @@ public function updateACL($data = null)
                             \CentreonDBInstance::getMonInstance()->query($request2);
                         }
                     } elseif ($data['action'] == 'DUP' && isset($data['duplicate_host'])) {
-                        // Get current configuration into Centreon_acl table
-                        $request = "SELECT group_id FROM centreon_acl " .
-                            "WHERE host_id = " . $data['duplicate_host'] . " AND service_id IS NULL";
+                        // Get current ACL configuration from centreon_storage.centreon_acl table
+                        $request = <<<SQL
+                            SELECT
+                                group_id
+                            FROM centreon_acl
+                            WHERE host_id = {$data['duplicate_host']}
```

Comment:
Could you make an extra effort to prepare the SQL query please ?

Reply:
Yep 👍 
https://github.com/centreon/centreon/pull/2213/commits/d48333813b7061af7fcf58f63ca8cc32ec85e45c